tests/datasource/data_connector/test_sql_data_connector.py

```diff
@@ -13,9 +13,10 @@
     sqlalchemy = pytest.importorskip("sqlalchemy")
 except ImportError:
     sqlalchemy = None
+import great_expectations.exceptions as ge_exceptions
 from great_expectations.validator.validator import Validator
 
-yaml = YAML()
+yaml = YAML(typ="safe")
```

Comment:
❤️ 

Perhaps we should do a quick grep through the codebase to make sure we've covered all of these? Perhaps a backlog item.

Reply:
Thank you :) GREAT-575 tracks this